src/drivers/net/bnxt/bnxt.c

```diff
@@ -2152,9 +2170,6 @@ static void bnxt_tx_adjust_pkt ( struct bnxt *bp, struct io_buffer *iob )
 	dbg_tx_vlan ( bp, ( char * )iob->data, prev_len, iob_len ( iob ) );
 	if ( iob_len ( iob ) != prev_len )
 		prev_len = iob_len ( iob );
-
-	iob_pad ( iob, ETH_ZLEN );
```

Comment:
The removal of TX padding seems to be unrelated to using the DMA APIs.  Is this change intentional?

Reply:
Restored this change, and will remove the unnecessary padding in a future commit